Northstar.CustomServers/mod/scripts/vscripts/_loadouts_mp.gnut

```diff
@@ -302,7 +310,7 @@ void function TryGivePilotLoadoutForGracePeriod( entity player )
 		else
 			respawnTimeReal = expect float( player.s.respawnTime )
 		
-		if ( ( ( Time() - respawnTimeReal <= CLASS_CHANGE_GRACE_PERIOD || GetGameState() < eGameState.Playing ) && file.loadoutGracePeriodEnabled ) || player.p.usingLoadoutCrate )
+		if ( ( ( Time() - respawnTimeReal <= CLASS_CHANGE_GRACE_PERIOD || GetGameState() < eGameState.Playing ) && file.loadoutGracePeriodEnabled ) || player.p.usingLoadoutCrate || ( file.allowChangeLoadoutFunc != null && file.allowChangeLoadoutFunc( player ) ) )
```

Comment:
I was requested on discord to change this to only use the callback function, and to have a default one that does the rest of this behaviour for the if statement. I personally disagree for the following reasons:

- Disabling the loadout grace period is its own function, so that behaviour should be separate from this new behaviour.
- Loadout crates should be always functional, and should imo be disabled by stopping the player from using them, as opposed to disabling their behaviour

Whilst i do disagree with the suggestion personally, I would like to know some more opinions on the matter, and am willing to make this change if needed

Reply:
only using the callback function seems weird to me